README.md

```diff
@@ -1,288 +1,49 @@
-# Bayesian Network Encounter Models
```

Comment:
The main README should include the intro material

Reply:
I am not sure how the main Readme got to this condition....